src/libDirectoryService/DSBlockPreProcessing.cpp

```diff
@@ -608,6 +723,11 @@ bool DirectoryService::DSBlockValidator(
     // [Sharding structure]
     curr_offset = PopulateShardingStructure(message, curr_offset);
 
+    if (!VerifyPoWOrdering())
+    {
+        LOG_GENERAL(INFO, "Failed to verify ordering");
```

Comment:
Need to return False here when enabling the shard verification.

Reply:
Resolved.